src/backend/server.js

```diff
@@ -3,9 +3,13 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 const path = require('path');
+const mcache = require('memory-cache');
 const db = require('./models/db');
 const seed = require('./models/seed/seed-db');
 
+app.set('view engine', 'jade');
```

Comment:
I believe this line is usually for setting view engines if you're using templates/frontend within Express - but our frontend is in React. Is there a different reason why this is here?